addons/portal/wizard/portal_wizard.py

```diff
@@ -33,6 +33,16 @@ def _default_partner_ids(self):
     partner_ids = fields.Many2many('res.partner', string='Partners', default=_default_partner_ids)
     user_ids = fields.One2many('portal.wizard.user', 'wizard_id', string='Users', compute='_compute_user_ids', store=True, readonly=False)
     welcome_message = fields.Text('Invitation Message', help="This text is included in the email sent to new users of the portal.")
+    has_invalid_emails = fields.Boolean('Are there users with invalid email', compute='_compute_has_invalid_emails', default=False)
+
+    @api.depends('user_ids.email_validity_state')
+    def _compute_has_invalid_emails(self):
+        for portal_wizard in self:
+            portal_wizard.has_invalid_emails = any(
```

Comment:
Could do:
```py
portal_wizard.has_invalid_emails = bool(portal_wizard.user_ids.filtered(
    lambda user: user.email_validity_state != 'valid'))
```

Your version works too, but we usually try to filter recordsets using "filtered".

EDIT: I just noticed that I'm the one who suggested to use "any" in the first place, oopsie, I changed my mind sorry :D